server/api/index.js

```diff
@@ -23,11 +36,35 @@ class Entity {
     }
   }
 
-  async update() {}
+  async update(req, res) {
```

Comment:
Explicitly define what should be updated, .e.g: 
`const { fieldsToUpdate} = req.body;`